tests/conftest.py

```diff
@@ -86,24 +198,65 @@ def test_account():
         configs.set("default_folder", f"/{current_username}")
 
 
-def load_and_share_resources(username_to_share):
+def load_and_share_resources(username_to_share, test_level):
     # Create the shared cluster using the test account
-    dotenv.load_dotenv()
+    if test_level in [TestLevels.UNIT, TestLevels.LOCAL]:
```

Comment:
I'm not sure there's a reason to do this on a remote cluster for higher test levels. In general we can test sharing and access from the test account in a local container, and if we find there are edge cases where the behavior is different from a cluster we can change it then. Also, I think this code should be going in a `test_account_loggedin_cluster` fixture which can be used by many tests. Actually I think there are a few rearrangements I would make here, let's chat about it on Mon.